pulsar-client-go/examples/consumer-listener/consumer-listener.go

```diff
@@ -0,0 +1,55 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package main
+
+import (
+	"../../pulsar"
+	"fmt"
+	"log"
+)
+
+func main() {
+	client := pulsar.NewClient().
+		ServiceUrl("pulsar://localhost:6650").Build()
```

Comment:
Not a hard or fast rule, but acronyms should be all caps. In some cases this can look terrible, but it is generally more idiomatic. So the option should be named `ServiceURL`. That said, is there any other kind of URL the client needs to set? If not, then `URL` should be just fine.

Reply:
yes, `URL` sounds better